workspace/apps/pidp/src/app/features/portal/portal.page.html

```diff
@@ -63,7 +63,7 @@
                   licence, then this step will become part of the process.</i
                 >
               </p>
-              <mat-stepper class="moa-message-steps">
+              <mat-stepper class="moa-message-steps" orientation="vertical">
```

Comment:
Excellent instinct to change the orientation, however Orientation="vertical" is now set for all layout views. We want to have it "horizontal" for the desktop, laptop, tablet views and "vertical" for mobile. 

We have 4 layout views that we develop for: `viewport-xsmall`, `viewport-small`, `viewport-medium` and `viewport-large`, which represent mobile, tablet, laptop and desktop views.